stix_shifter_modules/cbcloud/configuration/lang_en.json

```diff
@@ -16,11 +16,12 @@
     },
     "configuration": {
         "auth": {
-            "orgkey": {
+            "type" : "fields",
```

Comment:
Do you need the type key in the lang_en file? I'm not seeing it in the other connectors.

Reply:
ohh right. removed it